File: atividade_nicolas/alunos/nicolas_g2/giovana_lima/imc.py
# def acima(x,y,x):
#     if x < 19.8:
#         return 'Peso abaixo do ideal.'
#     elif x < 24.9:
#         return 'Peso ideal.'
#     else:
#         return 'Peso acima do ideal.'

def main():
    print('<-- Ajuste parâmetro IMC -->')
    altura = float(input('Insira sua altura em metros: '))
    peso = float(input('Insira seu peso em quilogramas: '))

    peso_ideal1 = 19.8 * altura**2
    peso_ideal2 = 24.9 * altura**2
    imc = peso / (altura*altura)

    peso_a_ganhar = peso_ideal1-peso
    peso_a_perder = peso-peso_ideal2

    if imc < 19.8:
        print(f'\nPeso abaixo do ideal.\nDeve ganhar: {(peso_a_ganhar):.2f}')
    elif imc < 24.9:
        print('\nPeso ideal.')
    else:
        print(f'\nPeso acima do ideal.\nDeve perder: {(peso_a_perder):.2f}')

    calorias_ingeridas = float(input('\nQuantas calorias você irá consumir em um dia?\n'))

    superavit = calorias_ingeridas + ((peso_a_ganhar * 1000)/90)
    deficit = calorias_ingeridas - ((peso_a_perder *1000)/90)

    if peso_a_perder > 0:
        print(f'\nGasto calórico por dia: {(deficit):.2f} cal')
    elif peso_a_ganhar > 0:
        print(f'\nGasto calórico por dia: {superavit:.2f} cal')
    else:
        print(f'\nPeso ideal. Não faz-se necessário alterações em dieta ou exercicios.')

if __name__ == '__main__':
    main()

File: atividade_nicolas/alunos/nicolas_g2/emanuel/imc.js
import {input} from "./io_utils.js"

function main(){

    const p = Number(input('Digite o seu peso: '))
    const a = Number(input('Digite a sua altura: '))

    function imc(){
        return (p / (a * a))
    }

    console.log(`IMC: ${imc().toFixed(1)}`)

    peso()

    function peso(){
        if(imc() < 19.8){
            console.log('Você deve ganhar peso para entrar na faixa de IMC normal')
    
            const ganhar_imc = 19.8 - imc()
    
            const kg = ganhar_imc * a * a
    
            console.log(`Você precisa ganhar ${kg.toFixed(3)} kg`)

            const consumo = Number(input('Digite quantas calorias você irá ingerir por dia: '))

            const cal = 7000 / kg

            const superavit = consumo - cal

            console.log(`Você deverá gastar ${superavit.toFixed(2)} calorias por dia`)
        }else if(imc() > 24.9){
            console.log('Você deve perder peso para entrar na faixa de IMC normal')
    
            const perder_imc = imc() - 24.9
    
            const kg = perder_imc * a * a
    
            console.log(`Você precisa perder ${kg.toFixed(3)} kg`)

            const consumo = Number(input('Digite quantas calorias você irá ingerir por dia: '))

            const cal = 7000 / kg

            const deficit = consumo + cal

            console.log(`Você deverá gastar ${deficit.toFixed(2)} calorias por dia`)
        }
    }
    
}
main()